resources/js/Directives/getinvolvedDirective.js

```diff
@@ -31,6 +31,21 @@ app.directive('getinvolved', ['$http', function ($http) {
             $scope.loadMore = function () {
                 $scope.totalDisplayed += 20;
             };
+
+            $scope.counter = function () {
+                if(self.contributors) {
+                    var keys = Object.keys(self.contributors);
```

Comment:
`self.contributors` is already an array.

`self.contributors.length` is enough.